content/docs/start/experiments.md

```diff
@@ -155,9 +158,9 @@ train:
 
 ## Tuning and running experiments
 
-We are definitely not happy with the `AUC` value, we got so far! Let's now tune
-and run the new experiment. Edit the `params.yaml` file to use bigrams and
-increase number of features:
+We are definitely not happy with the `AUC` value we got so far (`0.573`)! Let's
```

Comment:
let's not put precise values - it's hard to maintain (it's clear that you are never happy with it :) )

Reply:
Removed. I added it as a reminder as well, since that section is not immediately before this text. But I guess the user should still remember about AUC, at least if they read/follow this whole page at once.

Reply:
Should we link `[we got so far](#collecting-metrics)` though?